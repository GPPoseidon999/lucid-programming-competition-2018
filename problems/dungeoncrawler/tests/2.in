6
give up
up up 2 na you
never never 0
gon gon 1 never
give give 2 na gon
you you 0
na na 1 gon