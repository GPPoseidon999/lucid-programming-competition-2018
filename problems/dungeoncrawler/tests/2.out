never gon na give you up